e2e-tests/portal/cypress/e2e/journeys/checker/dashboard/dashboard.spec.js

```diff
@@ -124,7 +123,7 @@ context('View dashboard deals as a checker', () => {
     cy.get('table tr').eq(2).as('secondRow');
     const bssDealId = bssDeal._id;
 
-    cy.get('@secondRow').find(`[data-cy="deal__status--${bssDealId}"]`).should('exist');
+    dashboardDeals.rowIndex.status(2).should('exist');
```

Comment:
If row() the test cannot click on the deal on dashboard when creating deals through UI so we added `rowIndex()` to `e2e\pages\dashboardDeals.js` where the test is able to click on the deal on dashboard. @ttbarnes please advise if this is correct?

Reply:
Agree with @MarRobSoftwire , would be great to pass the index into `rowIndex` rather than child functions/selectors

If I remember correctly, we need to have both `rowIndex` and `row` until the dashboard tests are updated (separate PR).

Reply:
To be checked.